src/main/java/mate/academy/spring/config/AppConfig.java

```diff
@@ -0,0 +1,48 @@
+package mate.academy.spring.config;
+
+import java.util.Properties;
+import javax.sql.DataSource;
+import org.apache.commons.dbcp2.BasicDataSource;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.core.env.Environment;
+import org.springframework.orm.hibernate5.LocalSessionFactoryBean;
+
+@Configuration
+@PropertySource("classpath:application.properties")
+@ComponentScan(basePackages = "mate.academy.spring")
+public class AppConfig {
+    private final Environment environment;
+
+    @Autowired
+    public AppConfig(Environment environment) {
+        this.environment = environment;
+    }
+
+    @Bean
+    DataSource getDataSource() {
```

Comment:
add access modifiers
```suggestion
   public DataSource getDataSource() {
```